cfgov/v1/fragment_cache_extension.py

```diff
@@ -42,16 +48,15 @@ def parse(self, parser):
         return nodes.CallBlock(self.call_method('_cache_support', args),
                                [], [], body).set_lineno(lineno)
 
-    def _cache_support(self, name, timeout, caller):
+    def _cache_support(self, key, cache_name, timeout, caller):
         """Helper callback."""
-        key = self.environment.fragment_cache_prefix + name
```

Comment:
Because these changes remove the use of both `self.environment.fragment_cache_prefix` and `self.environment.fragment_cache`, is there a need any longer to set the defaults as is done in the `__init__` method [here](https://github.com/cfpb/cfgov-refresh/pull/4384/files#diff-c73c333b7f1696264ffedeb4d9d8a100R12)? I think maybe that custom `__init__` can be removed if those are no longer needed.

Reply:
@chosak Good call, removed in 441832558dcbc336174f0ad52b931f24dbd15b4e